frontend/components/site/SiteSettings/AdvancedSiteSettings.jsx

```diff
@@ -34,105 +34,90 @@ export const AdvancedSiteSettings = ({
+      <Field
+        name="engine"
+        id="engine"
+        component={p =>
+          <SelectSiteEngine
+            value={p.input.value}
+            onChange={p.input.onChange}
             className="form-control"
           />
-        </div>
-      </div>
+        }
+      />
+
+      {/* CUSTOM CONFIG */}
+      <fieldset>
+        <legend>Live site</legend>
+        <p className="well-text">
+          Add additional configuration in yaml to be added to your
+          {' '}
+          <code>_config.yml</code> file when we build your site&apos;s primary branch.
+        </p>
+        <label htmlFor="config">Site configuration</label>
+        <Field
+          component="textarea"
+          name="config"
+          className="form-control-mono"
+        />
+      </fieldset>
+
+      {/* DEMO CONFIG */}
+      <fieldset>
+        <legend>Demo site</legend>
+        <p className="well-text">
+          Add additional configuration in yaml to be added to your
+          {' '}
+          <code>_config.yml</code> file when we build your site&apos;s demo branch.
+        </p>
+        <label htmlFor="demoConfig">Demo configuration</label>
+        <Field
+          component="textarea"
+          name="demoConfig"
+          className="form-control-mono"
+        />
+      </fieldset>
+
+      {/* PREVIEW CONFIG */}
+      <fieldset>
+        <legend>Preview site</legend>
+        <p className="well-text">
+          Add additional configuration in yaml to be added to your
+          {' '}
+          <code>_config.yml</code> file when we build a preview branch for your site.
+        </p>
+        <label htmlFor="previewConfig">Preview configuration</label>
+        <Field
+          component="textarea"
+          name="previewConfig"
+          className="form-control-mono"
+        />
+      </fieldset>
+      <button
+        type="button"
+        className="usa-button usa-button-gray button-reset"
+        disabled={pristine}
+        onClick={reset}
+      >
+        Reset
+      </button>
+
+      <button
+        type="submit"
+        className="usa-button usa-button-primary"
+        disabled={pristine}
+      >
+        Save advanced settings
+      </button>
     </div>
 
-    <div className="usa-grid">
-      <div className="usa-width-one-whole">
-        <button
-          type="button"
-          className="usa-button usa-button-gray button-reset"
-          disabled={pristine}
-          onClick={reset}
-        >
-          Reset
-        </button>
-
-        <button
-          type="submit"
-          className="usa-button usa-button-primary"
-          disabled={pristine}
-        >
-          Save Advanced Settings
-        </button>
-      </div>
-    </div>
 
-    <div className="usa-grid">
-      <h3>Danger Zone</h3>
-      <div className="usa-alert usa-alert-delete" role="alert">
-        Delete this site from Federalist?
+    <div className="usa-alert usa-alert-error usa-alert-delete" role="alert">
```

Comment:
hrm, its role isn't really that of an alert though. Like this section isn't shown in response to a user action that triggered an error.

Using `usa-alert-*` classes was more to get the styling than for semantics.

Reply:
Yeah that makes sense, we need a `danger-well` ☠️  rather than an alert

Reply:
Good idea!

Reply:
Good point. I'm not opposed to using the alert styling if its cleaner but we shouldn't confuse folks with a "something is wrong" symbol.